om/om.go

```diff
@@ -56,40 +58,117 @@ func (o *OptimizedMultiLeaving) GetInterleavedRanking(num int, rks ...intergo.Ra
 	return cRks[maxIDx], nil
 }
 
-func (*OptimizedMultiLeaving) CalcInsensitivity(rks []intergo.Ranking, res []intergo.Res) float64 {
-	var iRkNum = len(rks)
-	var mean float64
+func getCredit(rankingIdx int, itemId int, idToPlacements *[]map[int]int, creditLabel int, isSameRankingIdx bool) float64 {
```

Comment:
Please use `idToPlacements []map[int]int` instead of `idToPlacements *[]map[int]int`

The reason is that Go's slice `[]` is already a reference type and  sometimes taking a pointer is expensive